blog-posts/sls-security-headers/sls-security-headers.md

```diff
@@ -0,0 +1,96 @@
+---
+published: false
+title: 'Leverage AWS CloudFront managed response headers policies'
+cover_image: 'https://dev-to-uploads.s3.amazonaws.com/uploads/articles/xgzowe15czab4awhanxd.png'
+description: ''
+tags: cloudfront, security, AWS, audit
+canonical_url:
+---
+
+– Did you just say security headers?
+
+– Yes.
+
+– Are you talking about CORS headers?
+
+– No.
+
+# HTTP security headers: a very simple addition to your website to efficiently prevent attacks
+
+I have learnt about HTTP security headers very late in my full stack developer carrier. In the last 5 years, have worked in a dozen teams to develop and run web applications.
+
+This is only very recently, when I joined [Theodo](https://www.theodo.fr/), that I was explained what HTTP security headers actually are. 😮
+
+HTTP security headers are seemingly unknown to many full stack developers.
+
+HTTP security headers are a mechanism to **protect your application from common attacks**. HTTP security headers are, as their name suggests, a set of HTTP headers with security purposes.
```

Comment:
Maybe add a line to explain that the headers are set by the server delivering the web pages, to be read by the browsers requesting them.

Reply:
J'ai l'impression qu'il y a un peu de repetition du terme 'HTTP security headers'. Pour alléger un peu tu peux faire qqc comme: `They are, as their name suggests, a set of HTTP headers with security purposes.`